scripts/src/main/resources/scripts/environment-project

```diff
@@ -23,6 +23,7 @@ function doLoadProperties() {
     then
       key="${key//$'\r'}"
       value="${value//$'\r'}"
+      echo "$key=$value"
```

Comment:
ups. Echo statements have to be removed. Thats why we create PRs and review (even if I review myself ;)).